Update README.md

teamsuperpanda opened the pull request:
HACS install instructions

pnbruckner commented:
Thanks, but when I suggested a change, I meant a change that makes this custom integration compatible with HACS. But you just suggested a change in the instructions. So are you now saying it actually is compatible with HACS?

I used to have specific instructions for using my custom components with HACS, but as I said, HACS' requirements kept changing and it became too difficult to keep up. So instead I just pointed at the HACS documentation (and even that has changed.) I don't intend to add back and try to maintain HACS-specific instructions.

But thanks anyway.

teamsuperpanda commented:
Turns out it is. Adding a custom repository works.
We are running the particular Pi on mobile and turns out between a poor connection and HACS selection I was getting errors.

I closed the issue. Then I went onto the HACS documentation to see what work needed to get done. I tried one last time to see exactly what the error was and it is working. I have deleted several times and added it back successfully. 

I understand HACS isn’t the ideal but I’m happy to help going forward.

By the way thank you for your work. Really enjoy this sensor.

pnbruckner commented:
You're welcome. I'm going to close this PR since, as I said, I don't want to maintain HACS specific instructions.
